src/cljc/witan/send/params.cljc

```diff
@@ -5,6 +5,8 @@
 
 (def some+ (fnil + 0))
 
+(def natural-prior 1/3)
```

Comment:
in a data or config file instead?

Reply:
Most of the clojurescript stuff is due to be retired now (theres a card for it) and shouldn't be being used in the main model, I've just checked as much as possible